pkg/cmd/controller/cmd.go

```diff
@@ -10,11 +13,25 @@ import (
 	"github.com/openshift/service-ca-operator/pkg/version"
 )
 
+var (
+	featureGates map[string]bool
```

Comment:
I'm a little concerned about future changes creating the potential for concurrent reads and writes of this map. Maybe you could instead declare this map within the call to `NewController` and replace `startServiceCAControllers` with a closure capturing it, rather than sharing state through this global?

Reply:
Good idea, reworked this with a local var and closure